layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -66,6 +66,12 @@ SRC_URI_append_photon-nano = " \
     file://tegra210-nano-cti-NGX003.dtb \
 "
 
+SRC_URI_append_photon-tx2-nx = " \
+    file://0001-cti-photon-merge-CDC-MBIM-driver-changes-from-bsp.patch \
```

Comment:
The CDC-MBIM driver changes patch is needed on this carrier board too? If so, we'll keep them.

Reply:
I think so, just validated a modem is working